examples/fanout-worker.e2e.ts

```diff
@@ -9,17 +9,14 @@ xdescribe('fanout-e2e', () => {
     const parentWorkflow: Workflow = {
       id: 'parent-workflow',
       description: 'simple example for spawning child workflows',
-      on: {
-        event: 'fanout:create',
-      },
```

Comment:
Shouldn't this be removed on the child instead of on the parent?

Reply:
Don't think it matters for this, but removed from both since we're calling `runWorkflow` directly.